src/App.tsx

```diff
@@ -1,26 +1,61 @@
-import React from "react";
-import logo from "./logo.svg";
-import "./App.css";
+import React, { Component } from "react";
+import "./App.scss";
 
-function App() {
-    return (
-        <div className="App">
-            <header className="App-header">
-                <img src={logo} className="App-logo" alt="logo" />
-                <p>
-                    Edit <code>src/App.tsx</code> and save to reload.
-                </p>
-                <a
-                    className="App-link"
-                    href="https://reactjs.org"
-                    target="_blank"
-                    rel="noopener noreferrer"
-                >
-                    Learn React
-                </a>
-            </header>
-        </div>
-    );
-}
+import { CssBaseline } from "@mui/material";
+import { OrchestratorForStories } from "./orchestrator/Orchestrator";
+import { ThemeProvider } from "lunatic-edt";
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+import frFile from "./i18n/fr.json";
+import Home from "./page/home/Home";
+
+i18n.use(initReactI18next).init({
+    resources: {
+        fr: {
+            translation: frFile,
+        },
+    },
+    lng: "fr",
+    fallbackLng: "fr",
+    interpolation: {
+        escapeValue: false,
+    },
+});
+
+type Source = {
+    source?: object;
+};
 
-export default App;
+export default class App extends Component<{}, Source> {
+    constructor(props: object) {
+        super(props);
+
+        this.state = {
+            source: undefined,
+        };
+    }
+
+    componentWillMount() {
```

Comment:
Even if you decide to make vintage React code, you really want to call a **Will** method?